_data/projects/PyConSys.yml

```diff
@@ -0,0 +1,12 @@
+name: PyConSys
+desc: Control System with Python and Artificial Intelligence (Evolutionary Algorithm)
+site: https://github.com/mck-sbs/PyConSys
+tags:
+  - python
+  - ai
+  - pid
+  - control-loop
+  - evolutional-algorithm
+upforgrabs:
+  name: help wanted
+  link: https://github.com/mck-sbs/PyConSys/projects/1
```

Comment:
@mck-sbs Please use this link `https://github.com/mck-sbs/PyConSys/labels/help%20wanted` for labels and also add issues in your Repo marked with this label.